website/docs/docs/collaborate/legacy-dbt-docs.md

```diff
@@ -0,0 +1,65 @@
+
+import SetUpDocsJob from '/snippets/_docs-setup-doc-job.md';
+
+<SetUpDocsJob/>
+
+Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.
+
+## Configure project documentation
+
+You configure project documentation to generate documentation when the job you set up in the previous section runs. In the project settings, specify the job that generates documentation artifacts for that project. Once you configure this setting, subsequent runs of the job will automatically include a step to generate documentation.
+
+1. Click the gear icon in the top right.
+2. Select **Account Settings**.
+3. Navigate to **Projects** and select the project that needs documentation.
+4. Click **Edit**.
+5. Under **Artifacts**, select the job that should generate docs when it runs.
+   <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/documentation-project-details.png" width="55%" title="Configuring project documentation"/>
+6. Click **Save**.
+
+## Generating documentation
+
+To generate documentation in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) run the `dbt docs generate` command in the **Command Bar** in the dbt Cloud IDE. This command will generate documentation for your dbt project as it exists in development in your IDE session.
+
+After generating your documentation, you can click the icon above the file tree, to see the latest version of your documentation rendered in a new browser window.
+
+## View documentation
+
+:::tip Use dbt Explorer for a richer documentation experience
+For a  richer and more interactive experience for understanding your project's resources and lineage, try out [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/).
+
+It includes map layers of your DAG, keyword search, interacts with the IDE, model performance, project recommendations, and more.
+:::
+
+Once you set up a job to generate [documentation](/docs/build/documentation) for your project, you can view documentation in the following ways:
+
+- In the dbt Cloud IDE, click the icon above the file tree in the editor to see the latest version of your documentation rendered in a new browser window. View your documentation for your dbt project while your code is still in development. With this workflow, you can inspect and verify what your project's generated documentation will look like before your changes are released to production.
+- In the dbt Cloud user interface, click **Documentation** in the navigation to view your project's documentation. This link will always help you find the most recent version of your project's documentation in dbt Cloud.
```

Comment:
This does not exist anymore! we should remove this bullet. docs are still available on the job run page i believe

Reply:
oh yea that's right, thank you `Dave!